src/api/serviceCalls.ts

```diff
@@ -1,19 +1,25 @@
 import axios from 'axios';
-import { employeeMetricsString, employeeDetailsString, allEmployeeMetricsString } from '@globals/constants';
+import { navigate } from 'gatsby';
+import { employeeMetricsString, employeeDetailsString, allEmployeeMetricsString, errorRoute, notFoundRoute } from '@globals/constants';
 
-export const getRequest = async (requestString: string, onSuccess: Function, onError: Function) => {
+export const getRequest = async (requestString: string, onSuccess: Function) => {
   await axios
     .get(requestString, { headers: { Authorization: `${axios.defaults.headers.common.Authorization}` } })
     .then((res) => {
       onSuccess(res.data);
     })
     .catch((err) => {
-      onError(err);
+      if (err.response.status === 404) {
```

Comment:
Looks like this is causing an error in one of the tests. Might just need to check for null / undefined status first.